driver/src/main/java/com/code_intelligence/jazzer/driver/FuzzTargetFinder.java

```diff
@@ -209,4 +213,39 @@ private static Optional<Method> targetPublicStaticMethod(
       return Optional.empty();
     }
   }
+
+  // Returns a list of all methods annotated with @FuzzTest without requiring @FuzzTest to be on the
+  // class path.
+  private static List<Method> findFuzzTests(Class<?> clazz) {
+    // Stores pairs of the form (method name, method descriptor).
+    HashSet<Map.Entry<String, String>> annotatedMethods = new HashSet<>();
+    // If an annotation is not on the classpath, it is silently ignored when the annotated element
+    // is loaded. We thus need to load and parse the class file ourselves.
+    try (InputStream stream = FuzzTargetFinder.class.getResourceAsStream(
```

Comment:
Is the `FuzzTargetFinder` class loader always able to lookup the given `Class` itself? If not the exception is thrown and the test simply fails, right?

Reply:
At this point it should be: `FuzzTargetRunner` looked up the class with its own class loader and found it, then passed the `Class` object to this method. Since `FuzzTargetFinder` is loaded by `FuzzTargetRunner`, I would expect it to find the class. That probably breaks with really weird class loaders (e.g. class loaders not based on `.class` files), but then I'm not sure we would support them that well elsewhere in Jazzer.

Reply:
I added the previous behavior as a fallback for this case.